CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 ## Unreleased
 
+### JMX metrics
+
+- Add support for newly named Tomcat MBean with Spring ([#1269])
```

Comment:
I don't think this kind of link works in the change log, see links below (also if you run `draft-change-log-entries.sh` it will help build this for you)

Reply:
Ok, good to know for next time. Sorry!